frontend/src/styles.scss

```diff
@@ -1,4 +1,5 @@
 @import "./app/libs/common-components/components/popover/popover.scss";
+@import "./app/libs/common-components/components/input/input-material.scss";
```

Comment:
also please remove this line, put it to correct place, but before all, merge latest master inside your branch